packages/indexer/src/indexers/autocrat/autocrat-proposal-indexer.ts

```diff
@@ -495,8 +495,8 @@ async function insertAssociatedAccountsDataForProposal(
     if (dao && daoDetails) {
       if (isQuote) {
         // Fail / Pass USDC
-        imageUrl = isFail
-          ? "https://imagedelivery.net/HYEnlujCFMCgj6yA728xIw/f38677ab-8ec6-4706-6606-7d4e0a3cfc00/public"
+        imageUrl = !isFail
```

Comment:
This is backwards now.

Reply:
that's the point

Reply:
bro this is the image it was using for fail:
![pUSDC](https://imagedelivery.net/HYEnlujCFMCgj6yA728xIw/f38677ab-8ec6-4706-6606-7d4e0a3cfc00/public)

Reply:
oh shit forget all I said